      
header:
  calibration_date: 2025-07-10
  frame_id: rs16_lidar
child_frame_id: top_camera
transform:
  translation:
    x: -0.43
    y: -0.013
    z: 0.46
  rotation: 
    x: -0.504502
    y: 0.503762
    z: -0.488671
    w: 0.502895